src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetFundsOut.sql

```diff
@@ -0,0 +1,14 @@
+﻿CREATE PROCEDURE [employer_financial].[GetFundsOut]
```

Comment:
It might be worth renaming the repo and sproc to match the type it returns i.e. `IPaymentFundsOutRepository` & `GetPaymentFundsOut`.